.github/workflows/quay-io-build-latest.yml

```diff
@@ -23,6 +24,9 @@ jobs:
         name: Build and push
         uses: docker/build-push-action@v2
         with:
-          context: openshift/system
+          context: .
+          file: openshift/system/Dockerfile
           push: true
-          tags: quay.io/3scale/porta:nightly
+          tags: |
+            quay.io/3scale/porta:latest
```

Comment:
Again just to consider, could this confuse anybody trying to pull latest stable build but get latest CI build instead?